### PR TITLE
Remove h-scroll on README panel

### PR DESCRIPTION
**Pull request by giles-v**
## Issue

When viewing text in the README panel, wide items like `<pre>` elements and tables can easily push the width of the document wider than the panel, making it difficult to read.

## Proposed Solution

A common fix for this issue (which manifests specifically in flex layouts) is to add `min-width: 0;` to the flex container. For this PR, I'm adding it to the `markdown-body` wrapper element, _as well as the panel container_, via `el.parentNode`. 

I know adjusting the style of the container _outside_ this component is bad practice. But:

- It achieves the purpose easily
- Making the same change "properly" will either mean an inscrutable PR to `storybook`, or to `storybook/addons` providing the ability to style panel containers - high lift for a small change
- If something goes wrong (such as the flex structure of storybook changing upstream) the worst case here is likely to be a simple reversion to previous behavior. 

In short, I think doing this is low risk and reasonable, and makes a big difference to the UX.

---

### Before - see the second paragraph is cut off:
screen shot 2018-05-22 at 11 32 05 am

### After - see the code container now scrolls instead of the whole body:
screen shot 2018-05-22 at 11 32 16 am

### CODE REVIEW

**Comment by ndelangen** [comment]
@tuchk4 when will be a good time to do a release do you think?

**Comment by tuchk4** [comment]
We can do it right now :)

I would like  to make next release with this one  https://github.com/tuchk4/storybook-readme/pull/60

**Comment by giles-v** [comment]
@tuchk4 any chance we can get a release while we wait for https://github.com/tuchk4/storybook-readme/pull/60? I'd love to get the feature in this PR available.
